packages/stylelint-plugin/src/index.ts

```diff
@@ -1,15 +1,15 @@
 import { Configuration } from "stylelint"
 
 const kaizenStylelintRules = {
-  "kaizen/prefer-var-tokens": [
+  "kaizen/no-invalid-use-of-var-tokens-in-functions": [
     true,
     { severity: "warning", disableFixing: false },
   ],
-  "kaizen/no-invalid-use-of-var-tokens-in-functions": [
+  "kaizen/no-invalid-use-of-var-tokens-in-equations": [
     true,
     { severity: "warning", disableFixing: false },
   ],
-  "kaizen/no-invalid-use-of-var-tokens-in-equations": [
+  "kaizen/no-deprecated-or-removed-tokens": [
```

Comment:
Moving this rule made the stylelint plugin work more reliably.
The order of the rules here affects which order they are applied in.

Reply:
This relates to the overarching issue with the PR: too much responsibility.
This reordering was because of the murkiness, which i'm trying to clear up.
Putting no-invalid-func before no-deprecated-or-removed-tokens meant that no-deprecated-or-removed-tokens wouldn't bail early because it detected an rgba function with an incompatible token.

Honestly, the splitting of these two rules was done purely because we didn't want no-deprecated-or-removed-tokens to be too un-cohesive. Now i kind of regret that as a premature attempt to make things cleaner.